src/developers/quick_start_guide.md

```diff
@@ -0,0 +1,55 @@
+This guide will take you through the steps of:
+- starting a `Notary` server
+- running a `Prover` to notarize some web data
+- running a `Verifier` to verify the notarized data
+
+Note that the TLSNotary protocol assumes that the `Notary` is trusted by the `Verifier`. To minimize the trust, the `Verifier` itself can act as a `Notary`.
+
+# Preliminaries
+
+### Install rust
+
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+# Guide
+
+### Start a Notary server:
+
+```sh
```

Comment:
`sh` is not in the list syntax highlighters: https://rust-lang.github.io/mdBook/format/theme/syntax-highlighting.html

`shell` can be used instead